tools/azure-sdk-tools/devtools_testutils/powershell_preparer.py

```diff
@@ -0,0 +1,79 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import functools
+import os
+
+from . import AzureMgmtPreparer
+from .resource_testcase import RESOURCE_GROUP_PARAM
+from azure_devtools.scenario_tests.exceptions import AzureTestError
+
+class PowerShellPreparer(AzureMgmtPreparer):
+    def __init__(
+        self, directory,
+        name_prefix='',
+        sku='Standard_LRS', location='westus', kind='StorageV2',
```

Comment:
Why do we need that in a generic preparer?

Reply:
We do not, removing them.